README.md

```diff
@@ -102,7 +104,7 @@ Which one of the following concepts does this best illustrate?
 What is the `ui-sref` directive, and how is it used?
 
 ```text
-Your answer goes here...
+ui-sref is equivalent to link_to in rails.  It establishes a link to the path or url that is referenced e.g. in the example above "<h1><a data-ui-sref="index">My App</a></h1>", "index" will be hyperlinked and when the user clicks on it, it will take them to 'My App' page
```

Comment:
I don't see any mention of state. It's important to indicate that it's loading a state (without refreshing), not an entirely new page.